api/src/main/java/com/datastrato/gravitino/annotation/Unstable.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Unstable APIs, with no guarantee on stability and compatibility across any releases. Classes that
+ * are unannotated are considered Unstable.
```

Comment:
> Classes that are unannotated are considered Unstable.

Should we employ code detection mechanisms to ensure that all classes in the API module should have annotations?

Reply:
I could not figure out a tool to check this, do you have any idea?

Reply:
It seems that detection can only be done through manually writing build tasks, which would complicate matters. 

Therefore, Maintaining the current status quo is also acceptable.